crates/services/consensus_module/poa/src/service.rs

```diff
@@ -129,7 +116,7 @@ pub(crate) enum RequestType {
 }
 
 pub struct MainTask<T, B, I> {
-    signing_key: Option<Secret<SecretKeyWrapper>>,
+    signer: SignMode,
```

Comment:
This should be a port we depend on instead of taking a direct dependency on `aws_sdk_kms`. I'll work on adding that.